README.md

```diff
@@ -99,15 +99,20 @@ This will provide autocomplete of component names/properties, as well as additio
 - Notepad or your favorite HTML editor
 - Web browser with access to the Internet
 
-### Testing
+### Deployment
 
-#### Gotchas
+1. Checkout the master branch. Your git state must be clean (no pending changes).
```

Comment:
Could you clarify what `Your git state must be clean (no pending changes).` means? Should the staging area, working tree or both be clean?